obspy/io/sac/header.py

```diff
@@ -22,7 +22,7 @@
                   when read into sac. [required]''',
        'b': 'F    Beginning value of the independent variable. [required]',
        'e': 'F    Ending value of the independent variable. [required]',
-       'iftype': '''I    Type of file [required]:
+       'iftype': '''I    Type of file [required]:\n
```

Comment:
Doesn't pass flake8:-( This just helps the docs look nicer.

________________________________
From: Elliott Sales de Andrade <notifications@github.com>
Sent: Friday, March 11, 2016 1:45:34 PM
To: obspy/obspy
Cc: Maccarthy, Jonathan K
Subject: Re: [obspy] SAC output ignores sampling rate from Trace metadata (#1317)


In obspy/io/sac/header.py<https://github.com/obspy/obspy/pull/1317#discussion_r55886099>:

> @@ -22,7 +22,7 @@
>                    when read into sac. [required]''',
>         'b': 'F    Beginning value of the independent variable. [required]',
>         'e': 'F    Ending value of the independent variable. [required]',
> -       'iftype': '''I    Type of file [required]:
> +       'iftype': '''I    Type of file [required]:\n


What is this escaped newline? Why just add a real newline?

-
Reply to this email directly or view it on GitHub<https://github.com/obspy/obspy/pull/1317/files#r55886099>.